javascript/packages/eslint-config-typescript/index.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+    extends: [
+        '@equisoft/eslint-config',
+        'plugin:@typescript-eslint/recommended',
+    ],
+    parser: '@typescript-eslint/parser',
+    settings: {
+        'import/parsers': {
+            '@typescript-eslint/parser': ['.ts', '.tsx'],
+        },
+    },
+    rules: {
+        'no-unused-vars': 'off',
+        '@typescript-eslint/no-unused-vars': ['error', { 'argsIgnorePattern': '^_$' }],
+
+        'no-useless-constructor': 'off',
+        '@typescript-eslint/no-useless-constructor': 'error',
+
+        'no-unused-expressions': 'off',
+        '@typescript-eslint/no-unused-expressions': 'error',
+
+        '@typescript-eslint/explicit-function-return-type': 'off', // See overrides
```

Comment:
Pourquoi faut que le mettes en override?

Reply:
Pour ça : https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md#configuring-in-a-mixed-jsts-codebase